packages/ffe-cards-react/src/IconCard/IconCard.tsx

```diff
@@ -35,13 +39,14 @@ function IconCardWithForwardRef<As extends ElementType>(
         >
             {({ CardAction }) => (
                 <>
-                    {React.cloneElement(icon, {
-                        ...icon.props,
-                        className: classNames(
-                            'ffe-icon-card__icon',
-                            icon.props.className,
-                        ),
-                    })}
+                    {!iconRight &&
```

Comment:
Vad med att gjøre noe slikt?

```
const content = [
    React.cloneElement(icon, {
        ...icon.props,
        key: 'icon',
        className: classNames(
            'ffe-icon-card__icon',
            icon.props.className,
        ),
    }),
    <div className="ffe-icon-card__body" key="children">
        {typeof children === 'function'
            ? children({...})
            : children}
    </div>
] 
    
 <>{iconRight ? content.reverse() : content}</>

```
Da slipper man ha icon koden 2 steder.

Reply:
Blir det lettere å lese? Dette ser ut som optimalisering for å spare noen karakterer i markup. Men leseligheten og vedlikeholdsgrad lider.

Reply:
Men jeg gjør et forsøk :-)

Reply:
Det er ikke før att spare karakterer egentligen. Det er før att slippa gjøra endringer på 2 steder hvis ikon koden skulle endre seg. Jag er open for andre forslag alltså. Kanske dele op ting mer i komponenter?